brozzler/worker.py

```diff
@@ -295,9 +295,13 @@ def brozzle_page(
     def _get_page_headers(self, page):
         # bypassing warcprox, requests' stream=True defers downloading the body of the response
         # see https://docs.python-requests.org/en/latest/user/advanced/#body-content-workflow
-        with requests.get(page.url, stream=True) as r:
-            page_headers = r.headers
-        return page_headers
+        try:
+            with requests.get(page.url, stream=True, verify=False) as r:
+                page_headers = r.headers
+                return page_headers
```

Comment:
it might be good to preserve the original indentation here

Reply:
Fixed. Thanks!